mava/components/component_groups.py

```diff
@@ -1,7 +1,12 @@
-from mava.components import executing
+from mava.components import executing, training
 
 # The components that are needed for a recurrent policy.
 recurrent_policy_components = [
     executing.RecurrentExecutorSelectAction,
     executing.RecurrentExecutorObserve,
 ]
+
+# The components that a needed for IPPO with the Huber value loss.
+huber_value_loss_components = [
+    training.HuberValueLossFunction,
```

Comment:
If there is just one component for the loss, should we be adding it to a "groups" file? 

Also, I think the group components here are general (since they are located inside the general components folder), so we can remove the string saying huber is just for IPPO, since other systems might want to use it as well.

Reply:
Agreed. I removed the IPPO text. I added the Huber component to a component group to standardise our component setup. But I don't have a strong opinion on whether it should be there or not. I'm happy to remove it if you think that is better :+1:

Reply:
I removed the component_group as we discussed :)